CHANGELOG.md

```diff
@@ -4,6 +4,17 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 
+v2.10.0
+------------------------------
+*December 4, 2018*
+
+### Added
+- `c-cuisinesWidget` component
+
+### Changed
+- `fozzie` module version update
+
+
 v2.9.0
 ------------------------------
 *November 28, 2018*
```

Comment:
Should remove the rounded corners from the `cuisines-widget.jpg`.  Guessing this should be handled by the CSS?

Reply:
@ashleynolan  Original assets for this block in the design are with rounded corners. Should I ask Bruno to upload them without them?

Reply:
I've uploaded version with straight corners(just cut the original image a little bit)

Reply:
👍